ml-agents/mlagents/trainers/sac/trainer.py

```diff
@@ -27,7 +29,14 @@ class SACTrainer(RLTrainer):
     """
 
     def __init__(
-        self, brain, reward_buff_cap, trainer_parameters, training, load, seed, run_id
+        self,
```

Comment:
Could you add type annotations on these?

Reply:
will do